Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -1177,6 +1177,21 @@ private extension LanguageServerProtocol.WorkspaceType {
   }
 }
 
+extension SourceKitLSPServer {
+  nonisolated func indexTaskDidProduceResult(_ result: IndexProcessResult) {
+    self.sendNotificationToClient(
+      LogMessageNotification(
+        type: result.failed ? .info : .warning,
```

Comment:
Just for testing? I wouldn't expect this to be particularly uncommon (eg. half way through making an edit and you save).

Also is this meant to be `result.failed ? .warning : .info`?